CMakeLists.txt

```diff
@@ -320,7 +320,9 @@ set_target_properties(ETISS PROPERTIES
   COMPILE_DEFINITIONS "ETISS_LIB_SHARED=1;ETISS_EXPORTS;ETISS_PLUGIN_EXPORTS"
   LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/lib/
 )
-TARGET_LINK_LIBRARIES(ETISS PUBLIC ${Boost_LIBRARIES} Boost::boost Threads::Threads)
+IF(UNIX)
```

Comment:
I think the removal of the previous line is the issue on windows.

However, it would make more sense to move this into an ELSE block below this conditional.

Reply:
oops sorry my bad..i messed up while resolving the conflict..i correct and try again and hope it works

Reply:
now it works..sorry for inconvenience..please merge it to master now